pkg/apiserver/certificate/certificate_test.go

```diff
@@ -28,7 +28,6 @@ import (
 	fakeclientset "k8s.io/client-go/kubernetes/fake"
 	certutil "k8s.io/client-go/util/cert"
 	"k8s.io/client-go/util/keyutil"
-
```

Comment:
Remove this line by accident?

Reply:
This empty line should be removed because the next import is not an antrea package.
This issue was found when changing code in this file but the change was removed later.

Reply:
You are right, I misread